doc/api/url.md

```diff
@@ -948,6 +948,24 @@ changes:
 The `url.format()` method returns a formatted URL string derived from
 `urlObject`.
 
+Example:
+
+```js
+const URL = require('url');
```

Comment:
Although it appears in other examples in this doc, I think in general we tend to leave out the `require()` for the module. I'm fine either way.

Reply:
Oh! But we do use `url` and reserve `URL` for the newer `url.URL`. So maybe remove it and use `url` rather than `URL` below.